core/tests/unit/ldml/keyboards/k_100_keytest.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+@@keys: [K_BKQUOTE]
+@@expected: \u0061
+-->
+
+<!--
+from https://github.com/unicode-org/cldr/blob/keyboard-preview/docs/ldml/tr35-keyboards.md#element-transform
+
+-->
+<!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
+<keyboard locale="en" conformsTo="techpreview">
+  <info author="srl295" indicator="🙀" layout="qwerty" normalization="NFC" />
+
+  <names>
+    <name value="TestKbd" />
+  </names>
+
+  <keys>
+    <key id="gap" gap="true" />
+    <key id="a" to="a" />
+  </keys>
+
+  <layers form="hardware">
+    <layer id="base">
+      <row keys="a gap"/>
+    </layer>
+  </layers>
+</keyboard>
+
+<!--
+  file:///Users/srl295/src/keymanapp/keyman/common/web/types/build/src/kvk/kvk-file-writer.js:62
+        str.len = value.length + 1;
+                        ^
+
+TypeError: Cannot read properties of undefined (reading 'length')
+    at KvkFileWriter.setString (file:///Users/srl295/src/keymanapp/keyman/common/web/types/build/src/kvk/kvk-file-writer.js:62:25)
+    at KvkFileWriter.build (file:///Users/srl295/src/keymanapp/keyman/common/web/types/build/src/kvk/kvk-file-writer.js:48:18)
+    at KvkFileWriter.write (file:///Users/srl295/src/keymanapp/keyman/common/web/types/build/src/kvk/kvk-file-writer.js:9:29)
+    at compileKeyboard (file:///Users/srl295/src/keymanapp/keyman/developer/src/kmc/build/src/kmc.js:73:31)
+    at file:///Users/srl295/src/keymanapp/keyman/developer/src/kmc/build/src/kmc.js:82:18
+    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
+    at async Promise.all (index 0)
+    at async ESMLoader.import (node:internal/modules/esm/loader:533:24)
+    at async loadESM (node:internal/process/esm_loader:91:5)
+    at async handleMainPromise (node:internal/modules/run_main:65:12)
+-->
```

Comment:
```suggestion
```

❓❓❓

Reply:
junk, will delete.

Reply:
this conversation is officially unresolvable :)

Reply:
💯